spec.html

```diff
@@ -31272,7 +31136,7 @@
               1. Append _kValue_ to the end of _kept_.
               1. Increase _captured_ by 1.
             1. Increase _k_ by 1.
-          1. Let _A_ be ? AllocateTypedArray(_C_, _captured_).
+          1. Let _A_ be ? Construct(_C_, «_len_»).
```

Comment:
Typo: len -> captured
And AllocateTypedArray ensures a new TypedArray instance is created, this is not the case with Construct.